src/library.js

```diff
@@ -448,16 +448,16 @@ LibraryManager.library = {
     }
     return ret;  // Previous break location.
   },
-
   system__deps: ['__setErrNo', '$ERRNO_CODES'],
-  system: function(command) {
-    // int system(const char *command);
-    // http://pubs.opengroup.org/onlinepubs/000095399/functions/system.html
-    // Can't call external programs.
-    ___setErrNo(ERRNO_CODES.EAGAIN);
-    return -1;
+  system: function(command) 
+  {
+	var cmd = Pointer_stringify(command);
+	var sys = require('util');
+	var exec = require('child_process').exec;
```

Comment:
the node.js specific code should be in a block `if (ENVIRONMENT_IS_NODE)`, that global var will be set if we are in node.

However, we shouldn't do it automatically, we should have a setting in settings.js, I think. The default behavior in node should be the same as on the web. but, we are adding an option for node to go directly to the underlying system